File: To_Lower_Case/answer.py

class Solution:

    def toLowerCase(self, str: str) -> str:

        rs = ""
        # 32
        section = ord("a") - ord("A")
        for s in str:
            if ord(s) >= ord("A") and ord(s) <= ord("Z"):
                rs = rs + chr(ord(s) + section)
            else:
                rs = rs + s
        return rs

sol = Solution()
print(sol.toLowerCase("Hello"))

File: To_Lower_Case/Answer.java
package tlc;

class Solution {

  public String toLowerCase(String str) {

    String rs = "";

    for (int i = 0; i < str.length(); i++) {
      char temp = str.charAt(i);
      if(temp >= 'A' && temp <= 'Z') {
        temp = (char) (temp + 32);
      }
      rs = rs + temp;
    }

    return rs;
  }
}

/**
 * Answer
 */
public class Answer {

  public static void main(String[] args) {
    String str = "Hello";
    Solution sol = new Solution();

    sol.toLowerCase(str);
  }
}
